docs/gather-job.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: insights-operator-job
+  annotations:
+    config.openshift.io/inject-proxy: insights-operator
+spec:
+  backoffLimit: 3
+  ttlSecondsAfterFinished: 600
+  template:
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: operator
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+        node-role.kubernetes.io/master: ""
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+        operator: Exists
+      - effect: NoExecute
+        key: node.kubernetes.io/unreachable
+        operator: Exists
+        tolerationSeconds: 900
+      - effect: NoExecute
+        key: node.kubernetes.io/not-ready
+        operator: Exists
+        tolerationSeconds: 900
+      volumes:
+      - name: snapshots
+        emptyDir: {}
+      - name: service-ca-bundle
+        configMap:
+          name: service-ca-bundle
+          optional: true
+      initContainers:
+      - name: insights-operator
+        image: quay.io/openshift/origin-insights-operator:latest
+        terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+        - name: snapshots
+          mountPath: /var/lib/insights-operator
+        - name: service-ca-bundle
+          mountPath: /var/run/configmaps/service-ca-bundle
+          readOnly: true
+        ports:
+        - containerPort: 8443
+          name: https
+        resources:
+          requests:
+            cpu: 10m
+            memory: 70Mi
+        args:
+        - gather
+        - -v=4
+        - --config=/etc/insights-operator/server.yaml
+      containers:
+        - name: sleepy
+          image: busybox
```

Comment:
Is this image available in the cluster registry or does it need to be downloaded? I guess the latter one....and that would be a bit problematic in an air-gapped cluster.

Reply:
it might be worth mentioning it in the docs for air-gaped users, but I don't know any simpler way to run anything on openshift :/